### PR TITLE
Cleanup dbi

### PR DESCRIPTION
**Pull request by alokmenghrajani**

### CODE REVIEW

File: server/src/main/java/keywhiz/ServiceModule.java [hunk 1]
```diff
@@ -146,79 +128,7 @@ public ServiceModule(KeywhizConfig config, Environment environment) {
     return DSL.using(dataSource.getConnection());
   }
 
-  // DBI
-
-  @Provides ObjectMapper configuredObjectMapper(Environment environment) {
```
Comment (sul3n3t) [inline server/src/main/java/keywhiz/ServiceModule.java]: Don't we still need the object mapper provides?

File: server/pom.xml [hunk 2]
```diff
@@ -93,10 +93,6 @@
       <groupId>io.dropwizard.modules</groupId>
       <artifactId>dropwizard-java8-auth</artifactId>
     </dependency>
-    <dependency>
```
Comment (sul3n3t) [inline server/pom.xml]: There's a matching entry w/ version in the root pom.xml

**Comment by sul3n3t** [comment]
LGTM aside from my two comments.

File: server/src/main/java/keywhiz/ServiceModule.java (hunk 1 above)
Comment (alokmenghrajani) [inline server/src/main/java/keywhiz/ServiceModule.java]: I moved it up a little.

**Comment by sul3n3t** [comment]
:+1:

**Pull request merged**